addons/hr_work_entry/static/src/components/work_entry_popover/work_entry_popover_service.js

```diff
@@ -115,7 +107,11 @@ export class WorkEntryPopoverService {
      */
     getSource(record) {
         for (const source of this._sources) {
-            if (record.data[source.field.name] && !source.expectedValue || record.data[source.field.name] && source.expectedValue === record.data[source.field.name]) {
+            if (
+                (record.data[source.field.name] && !source.expectedValue) ||
+                (record.data[source.field.name] &&
+                    source.expectedValue === record.data[source.field.name])
+            ) {
```

Comment:
```suggestion
            if (
                record.data[source.field.name] &&
                (
                    !source.expectedValue ||
                    source.expectedValue === record.data[source.field.name]
                 )
            ) {
```
I think this could be simplified a bit like this